packages/block-editor/src/hooks/test/__snapshots__/align.native.js.snap

```diff
@@ -19,25 +19,25 @@ exports[`Align options for group block sets Wide width option 1`] = `
 `;
 
 exports[`Align options for media block sets Align center option 1`] = `
-"<!-- wp:image {"align":"center","id":1,"sizeSlug":"large","linkDestination":"none"} -->
```

Comment:
Technically this test was incorrect before as it was testing the image block's align implementation instead of the hook, so it's nice that it's testing the correct thing now.